clients/shared/utils.go

```diff
@@ -44,7 +44,7 @@ func DefaultValue(column columns.Column, dialect sql.Dialect, additionalDateFmts
 			return nil, fmt.Errorf("colVal is not type *decimal.Decimal")
 		}
 
-		return val.Value(), nil
+		return val.String(), nil
```

Comment:
@Tang8330  This is the only (non-test) place we call `Decimal.Value()` I'm wondering if we can just call `.String()` instead?

Reply:
Especially since `.Value()` used to return a string when there was no precision or the precision was too high.